components/webui/imports/api/ingestion/server/CompressionDbManager.js

```diff
@@ -23,12 +23,12 @@ class CompressionDbManager {
      * Retrieves the last `limit` number of jobs and the ones with the given
      * job IDs.
      *
-     * @param {number} limit
+     * @param {string} lastUpdateDate
```

Comment:
If you agree it's better to pass in a timestamp, here we can write
```suggestion
     * @param {number} lastUpdateTimestamp in seconds.
```